modules/saml/lib/Message.php

```diff
@@ -221,9 +224,7 @@ public static function validateMessage(
 
         if (!$enabled) {
             return;
-        }
-
-        if (!self::checkSign($srcMetadata, $message)) {
+        } elseif (!self::checkSign($srcMetadata, $message)) {
```

Comment:
This change does not seem to do anything, why is it changed?

Reply:
For readability... No actual change